modules/oe_content_organisation/config/install/field.storage.node.oe_organisation_acronym.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
```

Comment:
acronym & logo are also present here https://github.com/openeuropa/oe_content/blob/EPIC-Project/modules/oe_content_entity/modules/oe_content_entity_organisation/config/install/
so we have 2 options:
1) since this is a content type separate from the organisation entity it could be better to keep the fields separate (requirements for the fields might split apart)
2) we create dedicated modules for acronym and logo and create a second PR for updating project ct

both have pros and cons so either is ok for me

Reply:
Yeah, I see what you mean, but it's better to have dedicated fields on the content type this time, so let's go for 1.